mw-webapp/src/component/header/Header.tsx

```diff
@@ -38,7 +39,23 @@ export const languageOptions: SelectItemType<Language>[] = [
 ];
 
 /**
- * Checkbox props
+ * Header type
+ */
+export enum HeaderType {
+
+  /**
+   * Important header
```

Comment:
Let's describe in the comments what disfferent between primary and secondary header

Reply:
fixed